tests/Tests.AzureAppConfiguration/Integration/IntegrationTests.cs

```diff
@@ -0,0 +1,544 @@
+using Azure;
+using Azure.Core;
+using Azure.Data.AppConfiguration;
+using Azure.Identity;
+using Azure.ResourceManager;
+using Azure.ResourceManager.AppConfiguration;
+using Azure.ResourceManager.AppConfiguration.Models;
+using Azure.ResourceManager.Resources;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManagement;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Tests.AzureAppConfiguration
+{
+    /// <summary>
+    /// Integration tests for Azure App Configuration that connect to a real service.
+    /// Creates a temporary App Configuration store for testing and deletes it after the tests are complete.
```

Comment:
How are stale stores cleaned up?

Reply:
I think the name of the resource group/store might need to be deterministic so that we can check for it on startup. This is for the case where `DisposeAsync` doesn't run and stale rgs/stores are left over.

Reply:
Discussed offline, will no longer be creating/cleaning up stores